.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,28 @@
+name: Prepare Buildpack Releases
+
+on:
+  workflow_dispatch:
```

Comment:
Have you triggered a workflow to make sure it runs correctly?

Reply:
it'll fail until #573 is sorted out